opensearch-operator/pkg/builders/cluster_test.go

```diff
@@ -233,5 +233,17 @@ var _ = Describe("Builders", func() {
 				SubPath:   oldKey,
 			}))
 		})
+
```

Comment:
Why the tests are uncommented?

Reply:
Hey @idanl21 I have added some new tests.